FileEmulationFramework/FileAccessServer.cs

```diff
@@ -117,6 +121,28 @@ private static int QueryInformationFileImpl(IntPtr hfile, IO_STATUS_BLOCK* ioSta
         }
     }
 
+    [UnmanagedCallersOnly(CallConvs = new[] { typeof(CallConvStdcall) })]
+    private static int QueryAttributesFileImpl(OBJECT_ATTRIBUTES* attributes, FILE_NETWORK_OPEN_INFORMATION* information)
+    {
+        lock (ThreadLock)
+        {
+            var result = _getFileAttributesHook.OriginalFunction.Value.Invoke(attributes, information);
+            var path = attributes->ObjectName->ToString().Substring(4); // Substring removes the "\??\" at the start of the path
```

Comment:
There's no specific hard documentation on which prefixes (device prefix, global prefix, etc.) are supported here.

Can you backport the String Trim from here? https://github.com/Reloaded-Project/reloaded.universal.redirector/blob/2230cf6c2525209bc23fd9f4b88b80b7f2e37cba/Reloaded.Universal.Redirector.Lib/Utility/Strings.cs#L39 and use that to sanitize the path.

Reply:
Done in 0589d76